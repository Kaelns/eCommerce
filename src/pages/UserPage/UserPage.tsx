import { Title } from '@/components/typography/Title/Title';
import styles from './UserPage.module.scss';
import UserProfile from '@/features/UserProfile/UserProfile';

// interface IProps {}

export function UserPage(/* props: IProps */): React.ReactNode {
  return (
    <>
      <Title className={styles.user}>User Profile Page</Title>
      <UserProfile />
    </>
  );
}
