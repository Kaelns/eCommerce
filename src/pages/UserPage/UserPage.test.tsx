import { render } from '@testing-library/react';

import { UserPage } from '@/pages/UserPage/UserPage';

describe('Given UserPage component', () => {
  it('When rendered, should match snapshot', () => {
    const { asFragment } = render(<UserPage />);

    expect(asFragment()).toMatchSnapshot();
  });
});
