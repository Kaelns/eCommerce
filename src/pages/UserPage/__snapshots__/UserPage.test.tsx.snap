// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Given UserPage component > When rendered, should match snapshot 1`] = `
<DocumentFragment>
  <h5
    class="MuiTypography-root MuiTypography-h5 _user_11f9a9 css-bezmlo-MuiTypography-root"
  >
    User Page
  </h5>
</DocumentFragment>
`;
